bentoml/_internal/frameworks/tensorflow_v2.py

```diff
@@ -263,23 +271,37 @@ def __del__(self):
     def _gen_run_method(runnable_self: TensorflowRunnable, method_name: str):
         raw_method = getattr(runnable_self.model, method_name)
         method_partial_kwargs = partial_kwargs.get(method_name)
-        if method_partial_kwargs:
-            raw_method = functools.partial(raw_method, **method_partial_kwargs)
-
-        def _mapping(item: "TFArgType") -> "tf_ext.TensorLike":
-            if not LazyType["tf_ext.TensorLike"]("tensorflow.Tensor").isinstance(item):
-                return t.cast("tf_ext.TensorLike", tf.convert_to_tensor(item))
-            else:
-                return item
 
         def _run_method(
             _runnable_self: TensorflowRunnable,
             *args: "TFArgType",
             **kwargs: "TFArgType",
         ) -> "ext.NpNDArray":
-            params = Params["TFArgType"](*args, **kwargs)
-            params = params.map(_mapping)
-            res = raw_method(*params.args, **params.kwargs)
+            try:
+                if method_partial_kwargs is None:
+                    res = raw_method(*args, **kwargs)
+                else:
+                    res = raw_method(*args, **dict(method_partial_kwargs, **kwargs))
```

Comment:
```suggestion
                    res = raw_method(*args, **kwargs, **method_partial_kwargs)
```

Reply:
They are different. 
`**kwargs, **method_partial_kwargs` may cause `TypeError: type object got multiple values for keyword argument`

Reply:
@sauyon may cause TypeError: type object got multiple values for keyword argument

```python
In [8]: def print_kw(**kwargs):
   ...:     print(kwargs)
   ...:

In [9]: print_kw(**{"a": 1}, **{"a": 2})
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-9-334143cf0bc0> in <module>
----> 1 print_kw(**{"a": 1}, **{"a": 2})

TypeError: print_kw() got multiple values for keyword argument 'a'

In [10]: print_kw(**dict({"a": 1}, **{"a": 2}))
{'a': 2}
```